src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/SearchController.php

```diff
@@ -65,4 +69,32 @@ public function indexAction(Request $request, AliasFilters $filters): Response
             ],
         ]);
     }
+
+    /**
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function searchAliasesForAssociationAction(Request $request): JsonResponse
+    {
+        try {
+            /** @var AliasForAssociation[] $searchTerms */
+            $searchTerms = $this->getQueryBus()->handle(new SearchAliasesForAssociation(
+                $request->get('query', ''),
+                (int) $request->get('limit', 20)
+            ));
+        } catch (AliasConstraintException $e) {
+            return $this->json([
+                'message' => $e->getMessage(),
+            ], Response::HTTP_BAD_REQUEST);
+        }
+
+        if (empty($searchTerms)) {
+            return $this->json([], Response::HTTP_NOT_FOUND);
+        }
+
+        return $this->json($searchTerms);
```

Comment:
```suggestion
        return ['aliases' => $this->json($searchTerms)];
```
Im suggesting this, so its much easier to add aditional params to response later. (for example if we need to add pagination, we could return "total" as well, without compromising the existing code)